### ISSUE TITLE
🆕 Software Suggestion | Swisscows

### ISSUE THREAD

**Issue by wuniversales** [privacytoolsIO/privacytools.io#1639]
Name: Swisscows
Category: Search Engine
URL: https://swisscows.com/

Safe search engine, no logs, no ip leak... (At least they say so)
https://swisscows.com/en/about#privacy-data

**Comment by LizMcIntyre** [comment]
I've tried Swisscows and have found the results to be good -- esp. with the symantic mapping feature. It's surprising how different the results can be with a few tweaks.

The privacy policy and other information looks promising, as you note, wuniversales. Would they be willing to answer the [Questions to Ask all Privacy Companies](https://www.reddit.com/r/privacytoolsIO/comments/dza44p/who_owns_your_favorite_privacy_service_part_2_the/)?

**Comment by smnthermes** [comment]
Suspicious. In every search there are results like this:

http://www.smartredirect.de/redir/clickGate.php?u=Pr6dLdBs&m=12&p=8PtMZHWq0t&q=ok%20boomer&url=https%3A%2F%2Fwww.vox.com%2F2019%2F11%2F19%2F20963757%2Fwhat-is-ok-boomer-meme-about-meaning-gen-z-millennials&r=https%3A%2F%2Fswisscows.ch%3Fquery%3Dok%20boomer

**Comment by smnthermes** [comment]
The domain `smartredirect.de` is included in some blocklists.

**Comment by nitrohorse** [comment]
Linking the [Discourse discussion](https://forum.privacytools.io/t/lets-discuss-swisscows-com-privacy-friendly-and-porn-free-search-engine-with-its-own-advertising-concept/) for reference. Notably their [privacy policy](https://swisscows.com/en/privacy) looks reasonable and they have a [unique company stance](https://swisscows.com/docs/Medienerziehung_EN.pdf) for creating a “family-friendly internet“ that in turn affects how they filter search results:
> 1. We promote moral values. 2. We hate violence and pornography. 3. We promote digital media education. - https://swisscows.com/en/about

**Comment by nitrohorse** [comment]
> The domain `smartredirect.de` is included in some blocklists.
> 
> Edit: when accessing it:
> 
> > This server belongs to adgoal ([www.adgoal.net](http://www.adgoal.net))

Yeah, looks like the top 3 items for search results are ad-based which have the redirects (swisscows.ch##.sales).
